projects/revaturecloud/selection/src/lib/complex-object-filter/complex-object-filter.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { RoomSearchQuery } from '../models/roomSearchQueryModel';
+
+@Component({
+  selector: 'lib-complex-object-filter',
+  templateUrl: './complex-object-filter.component.html',
+  styleUrls: ['./complex-object-filter.component.css']
+})
+export class ComplexObjectFilterComponent implements OnInit {
+
+  queryString: RoomSearchQuery;
+
+  constructor(queryString: RoomSearchQuery) {
+
+    this.queryString = queryString;
+  }
+
+  ngOnInit() {
+  }
+
+  complexObject() {
+
+    return (
```

Comment:
Directly change properties of a component rather than have a function that returns a value.